### ISSUE TITLE
[Accessibility] Links throughout webpage is not adapting Hyperlink theme in high contrast aquatic/desert theme.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#1843]
**User Experience:**
Visual impairment users will face difficulty if any item on the application would not adopt the default colors of high contrast black mode.

**Test Environment:**
OS: Win 10 Version 2004(OS Build 21296.1010)
URL: https://developer.microsoft.com/en-us/graph/graph-explorer 
Browser: Edge Dev

Prerequisite: Turn on the High contrast Aquatic/Desert Theme inside Settings>Accessibility>Contrast themes.

**Repro Steps:**

1. Open the above URL in Edge Dev.
2. Turn on the High contrast Aquatic/Desert mode setting.
3. Verify the links and observe the issue

_in desert mode_
![image](https://user-images.githubusercontent.com/18407044/174275694-9f99a556-0d3b-4dae-8691-7f1badc252f7.png)
_in aquatic mode_
![image](https://user-images.githubusercontent.com/18407044/174275886-e41f63c3-48f1-4b5f-bfcc-97cbfdceaf44.png)

Links impacted include:
1. Microsoft Graph API docs
2. Microsoft APIs Terms of Use
3. Microsoft Privacy Statement
4. @odata.nextLink links and other response query links
5. Microsoft Graph Toolkit links
6. Adaptive cards links
7. 'Open permissions panel' link
